lists exercise.md

```diff
@@ -1,21 +1,7 @@
-# Place an order on our online shop
+# Suggestions for the next GitHub demo
 
-To place an order on our online shop, follow these steps:
+It would be good to build off the basic understanding folks now have of working in GitHub. Some possible future demos could include using VS Code and GitHub Desktop, as well as any extensions for these that tech writers would find useful. I think it'd also be good to include an overview of DCOs. 
 
-1. Find the products you want to purchase
+Also, personally, I'd like to learn how to use git commands to commit and/or create pull requests as a way to kick my knowledge up a notch or two.
 
```

Comment:
This is my inline comment!

Reply:
Well done! More points to you, Anna :star: